lib/web/views/users/new.html

```diff
@@ -0,0 +1,24 @@
+{{ define "content" }}
+  {{ template "alert" . }}
+  <form class="form form--rounded-input form-registration" action="/users" method="post">
```

Comment:
Can we use `urlFor` helper to get the value for action?
```suggestion
  <form class="form form--rounded-input form-registration" action="{{ urlFor `users` `create` }}" method="post">
```